src/spec.ts

```diff
@@ -420,6 +420,7 @@ function normalizeOverlay(spec: UnitSpec, overlayWithPoint: boolean, overlayWith
         filled: true,
         role: 'pointOverlay'
       },
+      ...(selection ? {selection} : {}),
```

Comment:
If the selection is on the categories (not individual points) then selection over the line is sensible.   (Maybe we need to add more condition here.)